gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/client/PresenterWidget.java

```diff
@@ -225,6 +226,10 @@ public void fireEvent(GwtEvent<?> event) {
         getEventBus().fireEventFromSource(event, this);
```

Comment:
`public void fireEvent(GwtEvent<?> event)`

seems a bit redundant now.

Is there a web bindery replacement for HasHandlers?